coffeecard/CoffeeCard.Console.Refund/README.md

```diff
@@ -0,0 +1,11 @@
+﻿# Readme
+
+Simple ConsoleApp for refunding completed MobilePay orders. 
+
+## Dependencies
+The console app depends on `MobilePaySettings` from the ASP.NET `appsettings.json` file and a MobilePay certificate for communicating with the MobilePay Appswitch API.
+
+## How to run
+1. Insert OrderIds in the `input.txt` file, sepereated by a comma
```

Comment:
Why not have the input as an actual csv file? Then there wouldn't be any need for specifying the format of it?